client/packages/requisitions/src/utils.ts

```diff
@@ -68,9 +68,7 @@ export const isRequestDisabled = (request: RequestRowFragment): boolean => {
 
 export const isResponseDisabled = (response: ResponseRowFragment): boolean => {
   return (
-    response.status !== RequisitionNodeStatus.New ||
-    response.approvalStatus === RequisitionNodeApprovalStatus.Pending ||
-    response.approvalStatus === RequisitionNodeApprovalStatus.Denied
```

Comment:
how come removing these? can see duped logic in the `useIsResponseDisabled` hook, but looks like this util also used from list view?

Reply:
because approval status is None or null if they don't use auth...

Reply:
added back